libr/core/cmd_print.c

```diff
@@ -5877,7 +5877,7 @@ static int cmd_print(void *data, const char *input) {
 					(void)r_io_read_at (core->io, 0, data, core->offset);
 					char *res = r_print_json_path ((const char *)data, core->offset);
 					if (res) {
-						eprintf ("-> res(%s)\n", res);
+						R_LOG_ERROR ("-> res(%s)", res);
```

Comment:
-> res() looks more like a debug leftover than anything else. and this is clearly not an error. i think it will be better to replace with `r_cons_println (res);`